server/httpin/service.go

```diff
@@ -55,6 +55,8 @@ func NewService(proxy proxy.Service, appservices appservices.Service, conf confi
 	// Incoming remote webhooks.
 	h.HandleFunc(path.Webhook, h.Webhook).Methods(http.MethodPost)
 	h.HandleFunc(path.Webhook+"/{path}", h.Webhook).Methods(http.MethodPost)
+	h.HandleFunc(path.Webhook, h.WebhookValidate).Methods(http.MethodHead)
```

Comment:
Should there be a comment explaining why this special case exists?

Reply:
1/5 seems pretty self-explanatory to me, That's a typical use case for HEAD. However, should we [erform the secret validation, and if so perhaps `s/webhookValidate/webhookValidateAuthentication/` to better reveal the intention?